go/cmd/vtgr/main.go

```diff
@@ -15,15 +15,21 @@ package main
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/spf13/pflag"
 
 	"vitess.io/vitess/go/acl"
+	"vitess.io/vitess/go/vt/log"
 	"vitess.io/vitess/go/vt/servenv"
 	"vitess.io/vitess/go/vt/vtgr"
 )
 
+const deprecationMsg = "vtgr is deprecated and will be removed in Vitess 18. We recommend using VTOrc with semi-sync replication instead."
+
 func main() {
+	fmt.Println(deprecationMsg)
```

Comment:
Why do we need this? To print it in the help text? If yes, 👍

Reply:
Yep! `vtgr` is a standalone binary so you'll see the warning on the console anytime that you run it (even if you're only shown the --help output).